packages/react/src/components/Button/README.md

```diff
@@ -10,6 +10,16 @@ export default () => <Button>Content</Button>;
 
 ## Props
 
+#### `as`: String or Function
+
+```jsx
+export default () => <Button as="a" />;
+```
+
+```jsx
+export default () => <Button as={Link} />;
```

Comment:
Since `Link` is:
```
const Link = ({ children, to, ...props }) => (
  <a {...props} href={to}>
    {children}
  </a>
```
What is the difference between `a` and `Link`?

Reply:
It's just an example 😉
`Link` can be pulled out from `react-router-dom` for example

Reply:
Ohhhhh yup ok :) that makes sense :)